microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -22,145 +22,168 @@ ms.date: 05/04/2021
+1. Change the value of the `color` property of the `<h1>` element to your favorite color.  To do this, select the `<h1>` element in the **DOM Tree**. Select `fucshia`, type the new color, and then select it from the color list shown in the next screenshot.
 
-   ![The green indicator that the file is linked.](../media/workspaces-workspaces-demo-elements-styles-css.msft.png)
+   ![Change the color property in styles.css.](../media/workspaces-workspaces-demo-elements-styles-css-pick-color.msft.png)
 
-1. Open `styles.css` in a text editor again.  The `color` property is now set to your favorite color.
+1. Open `styles.css` in a text editor.  The `color` property is now set to the new color, which is orange in this example.
 
 1. Refresh the page.
 
-The color of the `<h1>` element is still set to your favorite color.  The change remains across a refresh, because when you made the change DevTools saved the change to disk.  And then, when you refreshed the page, your local server served the modified copy of the file from disk.
+The color of the `<h1>` element is still set to the new color.  The change remains across a refresh, because when you made the change, DevTools saved the change to disk. When you refreshed the page, your local server served the modified copy of the file from disk.
 
+**Tip:** You can also change the color by clicking the fucshia-colored swatch to open the color picker to pick a new color. The HEX value for the color you pick is the color name.
 
 <!-- ====================================================================== -->
-## Step 3: Save an HTML change to disk
+#### Save an HTML change to disk
 
-You can change HTML tagging using the Elements tool, but to be able to save the edits, we'll use the Sources tool.
+You can change HTML tagging using the **Elements** tool, but if you want to save your edits you need to use the **Sources** tool.
 
+<!---  Michael: following section was commented out in first version of doc. I uncommented it.  --->
+###### Try changing the HTML from the Elements panel
 
-### Change HTML from the Elements Panel
-
-You can make changes to the HTML content in the Element tool, but your changes to the DOM tree aren't saved to disk, and only affect the current browser session.
-
-The DOM tree is not the HTML source tagging.
-
-<!--### Try changing HTML from the Elements panel
+You can make changes to the HTML content using the **Elements** tool, but your changes to the DOM tree aren't saved to disk, and only affect the current browser session.
 
 > [!WARNING]
-> The workflow that you are about to try doesn't work.  You are trying it now so that you don't waste time later trying to figure out why it isn't working.
+> The workflow that you are about to try doesn't work. You are trying it now so that you don't waste time later trying to figure out why it isn't working.
```

Comment:
```suggestion
The following workflow demonstrates that edits aren't preserved across page refreshes, if you don't use a workspace.  We'll demonstrate this now so that you don't spend time later trying to figure out why it isn't working.
```